index.html

```diff
@@ -5,25 +5,143 @@
+      </p>
+      <p class="schedule"></p><div class="sched-text">2022.09.14 (Wed) ~ 18 (Sun)</div>
+        <div class="big">@ River Plate Stadium of Buenos Aires, Belgrano, etc.</div><p></p>
+    </div>
+  </section>
 
-  <section>
+  <section class="section2-wrapper">
+    <div class="section2">
+      <h2 class="myh1">Main Program</h2>
+      <hr class="hr2">
+      <div class="programs-wrapper">
+        <div class="program-block">
+          <img src="img/program_icon_01.png" width=64 height=58 alt="Presentation">
+          <div class="subtitle">Presentation</div>
+          <div class="description">The 10 teams will be presented together with their background and records, and key players.</div>
+        </div>
+        <div class="program-block">
+          <img src="img/program_icon_02.png" width=64 height=58 alt="The Games">
+          <div class="subtitle">The Games</div>
+          <div class="description">The 16 games divided into four days, with a minimum of four games per team will be played.</div>
+        </div>
+        <div class="program-block">
+          <img src="img/program_icon_03.png" width=64 height=58 alt="Press conference">
+          <div class="subtitle">Conference</div>
+          <div class="description">After each game two team members, one of them being the coach, will answer questions to the press.</div>
+        </div>
+        <div class="program-block">
+          <img src="img/program_icon_04.png" width=64 height=58 alt="Dissertation">
+          <div class="subtitle">Dissertation</div>
+          <div class="description">The closing day a dissertation by Marcelo Bielsa will precede the tournament game.</div>
+        </div>
+        <div class="program-block">
+          <img src="img/program_icon_05.png" width=64 height=58 alt="Awards">
+          <div class="subtitle">Awards</div>
+          <div class="description">After the closing game, the award ceremony will take place with a small convite for the participants and guests.</div>
+        </div>
+      </div>
+      <div class="texttoright">View Full Program ></div>
+    </div>
+  </section>
 
+  <section class="section3-wrapper">
+    <div class="section3">
+      <h2 class="myh1b">main teams</h2>
+      <hr class="hr2">
+      <div class="teams-wrapper">
+      </div>
+      <div class="texttoright2">See More Teams ></div>
+    </div>
   </section>
 
-  <section>
+  <hr class="hr3">
 
-  </section>
+  <section class="section4-wrapper">
+    <div class="section4">
+      <h2 class="myh1c">Sponsors</h2>
+      <hr class="hr2">
+      <div class="sponsor"><img src="img/mocst.png" width=320 height=120 alt="Ministry of Culture and Sports"></div>
+      <div class="sponsor"><img src="img/stackcommerce.png" width=320 height=120 alt="Stack Commerce"></div>
+      <div class="sponsor"><img src="img/macarthur.png" width=320 height=120 alt="Mc Arthur Foundation"></div>
 
+    </div>
+  </section>
+  
   <footer>
```

Comment:
- [ ] Footer is looking good, but let's use the Home design which is a light background footer with only one logo on the left and text on the right as the template shows:

Template: 
![Captura de pantalla 2022-04-14 a la(s) 10 10 50 p  m](https://user-images.githubusercontent.com/83717285/163512635-917f7139-ffdf-43c4-b784-2927ff6fb303.png)

Your design:
![Captura de pantalla 2022-04-14 a la(s) 10 10 11 p  m](https://user-images.githubusercontent.com/83717285/163512665-b7d14124-b84b-4021-b57e-b4fcf80a2aac.png)

Reply:
@smunozmo: I took this footer from the live version my mentor passed me (http://summit.cckorea.org/main.html) and I followed it with great detail. (as other things you pointed out, oh I can't believe it, it's different from the official template!! :rofl: :rofl:) Can I leave my version as it is? Please :pray:

Reply:
Hi @acolombo1, I don't it's possible to have a different design from the original template as it is a requirement for every student, I'm sorry it looks great! But it should follow the design guidelines.